doc/api/os.md

```diff
@@ -33,205 +42,285 @@ defined are described in [OS Constants][].
-Windows.
+The load average is a UNIX-specific concept with no real equivalent on
+Windows platforms. On Windows, the return value is always `[0, 0, 0]`.
 
 ## os.networkInterfaces()
 <!-- YAML
 added: v0.6.0
 -->
 
-Get a list of network interfaces:
+The `os.networkInterfaces()` method returns an object listing only network
+interfaces that have been assigned a network address.
+
+Each key on the returned object identifies a network interface. The associated
+value is an array of objects that each describe an assigned network address.
+
+The properties available on the assigned network address object include:
+
+* `address` {String} The assigned IPv4 or IPv6 address
+* `netmask` {String} The IPv4 or IPv6 network mask
+* `family` {String} Either `IPv4` or `IPv6`
+* `mac` {String} The MAC address of the network interface
+* `internal` {boolean} `true` if the network interface is a loopback or
+  similar interface that is not remotely accessible; otherwise `false`
+* `scopeid` {number} The numeric IPv6 scope ID (only specified when `family`
+  is `IPv6`)
 
 ```js
-{ lo:
-   [ { address: '127.0.0.1',
-       netmask: '255.0.0.0',
-       family: 'IPv4',
-       mac: '00:00:00:00:00:00',
-       internal: true },
-     { address: '::1',
-       netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
-       family: 'IPv6',
-       mac: '00:00:00:00:00:00',
-       internal: true } ],
-  eth0:
-   [ { address: '192.168.1.108',
-       netmask: '255.255.255.0',
-       family: 'IPv4',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false },
-     { address: 'fe80::a00:27ff:fe4e:66a1',
-       netmask: 'ffff:ffff:ffff:ffff::',
-       family: 'IPv6',
-       mac: '01:02:03:0a:0b:0c',
-       internal: false } ] }
+{
+  lo: [
+    {
+      address: '127.0.0.1',
+      netmask: '255.0.0.0',
+      family: 'IPv4',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    },
+    {
+      address: '::1',
+      netmask: 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff',
+      family: 'IPv6',
+      mac: '00:00:00:00:00:00',
+      internal: true
+    }
+  ],
+  eth0: [
+    {
+      address: '192.168.1.108',
+      netmask: '255.255.255.0',
+      family: 'IPv4',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    },
+    {
+      address: 'fe80::a00:27ff:fe4e:66a1',
+      netmask: 'ffff:ffff:ffff:ffff::',
+      family: 'IPv6',
+      mac: '01:02:03:0a:0b:0c',
+      internal: false
+    }
+  ]
+}
 ```
 
-Note that due to the underlying implementation this will only return network
-interfaces that have been assigned an address.
-
 ## os.platform()
 <!-- YAML
 added: v0.5.0
 -->
 
-Returns the operating system platform. Possible values are `'darwin'`,
-`'freebsd'`, `'linux'`, `'sunos'` or `'win32'`. Returns the value of
-[`process.platform`][].
+The `os.platform()` method returns a string identifying the operating system
+platform as set during compile time of Node.js.
+
+Currently possible values are:
```

Comment:
According to [the list of valid operating systems in `configure`](https://github.com/nodejs/node/blob/84ad31fff369f0ec9a972c2a11cdefee316dab9d/configure#L30-L31) and the name changes in `node.gyp`, this list should currently be:

* aix
* android
* darwin
* freebsd
* linux
* openbsd
* sunos
* win32